splunk_eventgen/eventgen_core.py

```diff
@@ -1,33 +1,21 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # encoding: utf-8
 import imp
+import importlib.util
 import logging
 import logging.config
 import os
 import sys
 import time
 import signal
-from Queue import Empty, Queue
+from queue import Empty, Queue
 from threading import Thread
 
-from lib.eventgenconfig import Config
-from lib.eventgenexceptions import PluginNotLoaded
-from lib.eventgentimer import Timer
-from lib.outputcounter import OutputCounter
-from lib.logging_config import logger
-
-lib_path_prepend = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'lib')
-sys.path.insert(0, lib_path_prepend)
-# Since i'm including a new library but external sources may not have access to pip (like splunk embeded), I need to
-# be able to load this library directly from src if it's not installed.
-try:
-    import logutils
-    import logutils.handlers
-except ImportError:
-    path_prepend = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'lib', 'logutils_src')
-    sys.path.append(path_prepend)
-    import logutils
-    import logutils.queue
```

Comment:
Change relative imports to absolute imports and remove all related path inserts.